django/db/backends/base/schema.py

```diff
@@ -7,6 +7,8 @@
 from django.utils import timezone
 from django.utils.encoding import force_bytes
 
+from . import references
```

Comment:
Do you prefer `references.X` as opposed to importing each class? We've sometimes removed that pattern elsewhere.